docs/pages/guides/using-firebase.mdx

```diff
@@ -123,6 +123,7 @@ const { getDefaultConfig } = require('@expo/metro-config');
 
 const defaultConfig = getDefaultConfig(__dirname);
 defaultConfig.resolver.sourceExts.push('cjs');
+config.resolver.unstable_enablePackageExports = false;
```

Comment:
```suggestion
const config = getDefaultConfig(__dirname);
config.resolver.sourceExts.push('cjs');
config.resolver.unstable_enablePackageExports = false;
```

There's a typo in the suggested change. The variable assigned was `defaultConfig` but let's change it to `config` due to the default metro config suggestions from its reference: https://docs.expo.dev/versions/latest/config/metro/#packagejsonexports. Otherwise `config` will throw an error.